### PR TITLE
added in templates Post and Term field data receiving

### PR DESCRIPTION
**Pull request by mzhirov**

### CODE REVIEW

File: views/employee/archive-department.php [hunk 1]
```diff
@@ -16,6 +21,19 @@
...
+			<?php
+			// print custom field department, if it exists
+			if ( $fields->department ) : ?>
+				<h3><?php echo $fields->department; ?></h3>
+			<?php endif; ?>
+			<?php
+			// print custom field floor, if it exists
+			if ( $fields->floor ) : ?>
+				<h3><?php echo $fields->floor; ?></h3>
+			<?php endif; ?>
```
Comment (aprokopenko) [inline views/employee/archive-department.php]: I think for taxonomy archive more actual fields will be like some featured image and subheading.

fields like "floor" and "department" doesn't mean anything for grouped members page

File: views/employee/single.php [hunk 2]
```diff
@@ -3,11 +3,28 @@
...
+use JustCoded\WP\Framework\Objects\Postmeta;
+
 /* @var \JustCoded\WP\Framework\Web\View $this */
 
+// get custom fields data
+$fields = new Postmeta();
+
 $this->extends( 'layouts/main' ); ?>
 
 	<h1><?php the_title(); ?></h1>
```
Comment (aprokopenko) [inline views/employee/single.php]: the_title() can't be used outside of the loop (same for field values).

so this should be added inside  employee/_content template (that is where you print real data)

**Review by aprokopenko — CHANGES_REQUESTED** [review]
please make more accurate examples

File: views/employee/archive-department.php (hunk 1 above)
Comment (mzhirov) [reply views/employee/archive-department.php]: renamed fields

File: views/employee/single.php (hunk 2 above)
Comment (mzhirov) [reply views/employee/single.php]: replaced

**Pull request merged**